src/services/types.ts

```diff
@@ -0,0 +1,20 @@
+import { Json } from "@polkadot/types";
+import React from "react";
+
+export interface SchemaDetails {
+  key: React.Key;
+  schema_id: string;
+  model_type: string;
+  payload_location: string;
+  model_structure: Json;
+}
+
+export interface MessageDetails {
+  key: React.Key;
+  payload: Json;
+  payload_length: string;
+}
+
+export interface SchemaProps {
```

Comment:
This one I would say remove all together however. See comments on the two components that use it.